modules/ROOT/partials/configuration/exportword.adoc

```diff
@@ -71,14 +95,18 @@ tinymce.init({
   plugins: 'exportword',
   toolbar: 'exportword',
   exportword_service_url: '<service_URL>', // required
-  exportword_converter_options: { // required to support "exportword_converter_style"
-    header: [
-      {
-        html: '<h1>First page header.</h1>', //example
-        css: 'h1 { font-size: 30px; }', //example
+  exportword_converter_options: {
```

Comment:
On line 83 (a few lines above this comment), can maybe add a note to say that the header and footer have separate styling and this option doesn't apply to them.

Reply:
Added note in [commit](https://github.com/tinymce/tinymce-docs/pull/3390/commits/10999a7875618623b5d74258bbc9647a5008c9b3) for option description rather than the config example.